apps/camera/js/views/hud.js

```diff
@@ -40,14 +40,37 @@ module.exports = View.extend({
     return this;
   },
 
+  _setLabel: function(element, mode) {
+    if (mode) {
+      this.els[element].setAttribute('data-l10n-id', mode.icon + '-button');
```

Comment:
I suggest you add new data to the underlying `config.js` that is dedicated to mapping a flash/camera mode to an `l10n-id`.